avro/src/test/java/com/fasterxml/jackson/dataformat/avro/interop/annotations/AvroEncodeTest.java

```diff
@@ -74,8 +76,67 @@ protected CustomComponent read(Object reuse, Decoder in) throws IOException {
 
     }
 
-    protected Wrapper wrapper;
+    public static class UuidAsBytesAvroEncoding extends CustomEncoding<UUID> {
```

Comment:
The original format of this test meant that if the custom deserializer wasn't picked up, Jackson was still flexible enough that it would pass the test because it could read what it wrote. This `UuidAsBytesAvroEncoding` actually changes the serialized format, which means this will now break if the custom serializers/deserializers are ignored.